index.html

```diff
@@ -633,16 +633,75 @@ <h3>Multiple Schemas</h3>
         <section class="informative">
           <h3>Validity of a Verifiable Credential</h3>
           <p>
-            Validation against a [[JSON-SCHEMA]] may be confused with 
+            Validation against a [[JSON-SCHEMA]] may be confused with
             <a href="https://www.w3.org/TR/vc-data-model/#dfn-credential-validation">validation</a>
             or <a href="https://www.w3.org/TR/vc-data-model/#dfn-verify">verification</a>
             of a Verifiable Credential. A valid credential according to a [[JSON-SCHEMA]] refers
             only to the structure of the claims comprising a Verifiable Credential. This idea of
             validity does not imply anything about the validity of the Verifiable Credential itself.
-            It's possible for a Verifiable Credential to be considered valid by one verifier, 
+            It's possible for a Verifiable Credential to be considered valid by one verifier,
             while another verifier would not consider it valid.
           </p>
         </section>
+        <section class="informative">
+          <h3>Relationship to Verifiable Credential Type Property</h3>
+          <p>
+            It is common to define a <code>credential schema</code> that will be set for Verifiable Credentials whose <a href="https://www.w3.org/TR/vc-data-model/#dfn-type">type</a>
+            property contains a specific <code>type</code>. In this scenario, it is RECOMMENDED to use the value of the specific type as the name property of
```

Comment:
There is a general guidance to avoid RFC2119 language in informative sections. Until/unless the WG decides to ignore that guidance, we should adhere to it, and change from `RECOMMENDED` to `advised` or similar.

Reply:
https://github.com/w3c/vc-json-schema/pull/139